salt/modules/rpm.py

```diff
@@ -168,14 +168,34 @@ def verify(*packages, **kwargs):
               'g': 'ghost',
               'l': 'license',
               'r': 'readme'}
+    options_map = {
+        'nocaps': '--nocaps',
+        'nodeps': '--nodeps',
+        'nodigest': '--nodigest',
+        'nofiledigest': '--nofiledigest',
+        'nofiles': '--nofiles',
+        'nogroup': '--nogroup',
+        'nolinkto': '--nolinkto',
+        'nomode': '--nomode',
+        'nomtime': '--nomtime',
+        'nordev': '--nordev',
+        'noscripts': '--noscripts',
+        'nosignature': '--nosignature',
+        'nosize': '--nosize',
+        'nouser': '--nouser'
+    }
     ret = {}
     ignore_types = kwargs.get('ignore_types', [])
+    verify_options = kwargs.get('verify_options', [])
+    cmd = ['rpm']
+    for option in verify_options:
+        cmd.append(options_map.get(option, ''))
```

Comment:
This is not a good idea, if the passed option is not in the the map then there will be an empty string appended to the list of arguments passed to cmd.run. This list of arguments will be passed to ``subprocess.Popen()`` which, depending on the command being run, may not work as expected.

Also, the map seems completely unnecessary since what we end up doing is prepending two dashes.

Most options in salt which we expect to be a list, we also allow to be passed as a comma-separated list, so we should be splitting on a list if this is a string.

I'll merge this and add some fixes.